DEPLOY_LINUX.md

```diff
@@ -0,0 +1,87 @@
+
+## Deploy the Console on Linux
+
+The console can be bound to the same web listener as the controller's management API. The console is then accessed by navigating to the controller's web address.
+
+1. On the controller host, download the latest release of the console.
+
+    ```text
+    wget https://github.com/openziti/ziti-console/releases/latest/download/ziti-console.zip
+    ```
+
+1. Unzip the console in the controller's working directory.
+
+    ```text
+    sudo unzip -d /var/lib/ziti-controller/zac ./ziti-console.zip
+    ```
+
+1. In the controller's configuration file, locate the web listener that binds the `edge-management` API. You must bind the console to the same listener.
```

Comment:
too much detail imo. simply show them the final stanza and replace the ./zac with either the full path to where the zac was unzipped
```
      - binding: zac
        options: { "location": "./zac", "indexFile":"index.html" }
```